net-misc/i2pd/i2pd-2.6.0-r3.ebuild

```diff
@@ -53,27 +53,38 @@ src_configure() {
 
 src_install() {
 	cmake-utils_src_install
+
+	# config
+	insinto "/etc/i2pd"
+	doins "${S}/docs/i2pd.conf"
+	doins "${S}/debian/subscriptions.txt"
+	doins "${S}/debian/tunnels.conf"
+
+	# doc
 	dodoc README.md
+
+	# working directory
 	keepdir /var/lib/i2pd/
-	insinto "/var/lib/i2pd"
+	insinto /var/lib/i2pd
 	doins -r "${S}/contrib/certificates"
 	dosym /etc/i2pd/subscriptions.txt /var/lib/i2pd/subscriptions.txt
 	fowners "${I2PD_USER}:${I2PD_GROUP}" /var/lib/i2pd/
 	fperms 700 /var/lib/i2pd/
-	dodir "/etc/${PN}"
-	insinto "/etc/${PN}"
-	doins "${S}/docs/${PN}.conf"
-	doins "${S}/debian/subscriptions.txt"
-	doins "${S}/debian/tunnels.conf"
```

Comment:
We made them belong in ```/etc/i2pd```, as
- this is the place where configs belong
- they fall under ```CONFIG_PROTECT```, in case of updates.
```i2pd``` itself searches in in ```~/.i2pd/``` as well as ```/var/lib/i2pd/```, which is misbehavior inherited from the original i2p java software. As (co-)maintainer of I2P I never cared much about it, though. These are configuration files, meant to be edited. Not state info.

A copy, however, of these files in ```/usr/share/i2pd/doc``` might be an idea, so the user has easy access to "virgin" files. Do we want that?